.github/workflows/compile.yml

```diff
@@ -23,7 +23,6 @@ jobs:
         # fake committer details to satisfy git am
         git config --global user.name "No One"
         git config --global user.email "noone@nowhere.com"
```

Comment:
These lines are no longer needed if we're not doing the `patch_sail_riscv` thing.